Controller/Admin/ModuleSupport.php

```diff
@@ -0,0 +1,195 @@
+<?php
+/**
+ * Shop System Plugins:
+ * - Terms of Use can be found under:
+ * https://github.com/wirecard/oxid-ee/blob/master/_TERMS_OF_USE
+ * - License can be found under:
+ * https://github.com/wirecard/oxid-ee/blob/master/LICENSE
+ */
+
+namespace Wirecard\Oxid\Controller\Admin;
+
+use OxidEsales\Eshop\Application\Controller\Admin\AdminController;
+use OxidEsales\Eshop\Core\Registry;
+use OxidEsales\Eshop\Core\Module\Module;
+use Wirecard\Oxid\Core\Helper;
+use Exception;
+
+/**
+ * Controls the view for the Module support tab in Module detail.
+ */
+class ModuleSupport extends AdminController
+{
+    /**
+     * @inheritdoc
+     *
+     */
+    protected $_sThisTemplate = 'module_support.tpl';
+
+    /**
+     * current module
+     *
+     * @var OxidEsales\Eshop\Core\Module\Module
+     */
+    protected $_oModule;
+
+    /**
+     * @inheritdoc
+     *
+     * @return string
+     */
+    public function render()
+    {
+        $sModuleId = $this->getEditObjectId();
+        $sDefaultEmail = $this->getConfig()->getActiveShop()->oxshops__oxinfoemail->value;
+        $this->_aViewData['oxid'] = $sModuleId;
+        $this->_aViewData['contact_email'] = $this->_getModule()->getInfo('email');
+        $this->_aViewData['default_email'] = $sDefaultEmail;
+        $this->_aViewData['should_be_rendered'] = Helper::isThisModule($sModuleId);
+        if (empty($this->_aViewData['from_email'])) {
+            $this->_aViewData['from_email'] = $sDefaultEmail;
```

Comment:
This value is only used for pre-filling the form field input from the Mater Settings of OXID. From the user story: _As a merchant/integrator I can enter a sender address_. Now merchant can enter his own, but doesn't have to because it is already pre-filled.

Reply:
It is used only as pre-filled value. User can change it.